lib/runner/request-helpers-presend.js

```diff
@@ -381,5 +383,31 @@ module.exports = [
             }
             done();
         });
+    },
+    // Raw request body update
+    function (context, run, done) {
+        if (!context.item) { return done(new Error('Nothing to update body.')); }
+
+        let request = _.get(context.item, 'request'),
+            requestBody = _.get(request, 'body'),
+            bodyParams;
```

Comment:
Can we use ```lib/runner/request-helpers-presend.js``` as source for removing JSON comments and leave core body builder untouched ? @codenirvana  any thoughts on this ?